### DISCUSSION TITLE
Investiny within Investpy

### DISCUSSION THREAD

**Discussion by mrSnook-swe** [alvarobartt/investiny#48] [Ideas]
Hello!
I like investiny alot. Great work!
In the ambition of transferring everything to investpy in the future. Is it possible to keep investiny as a core in investpy?
I e to still provide access to the interfaces search_assets and historical_data the way they work in investiny.

**Comment by alvarobartt** [comment]
Sure, ideally I'll just move this code to `investpy` first and then I'll add some `FutureDeprecationWarning` messages so as to let everyone know that the current `investpy` functionality will be legacy in the next release! Thanks for the idea :hugs:
